geode-for-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSet.java

```diff
@@ -512,22 +505,27 @@ public long zrevrank(byte[] member) {
   public static long zunionstore(RegionProvider regionProvider, RedisKey key,
       List<ZKeyWeight> keyWeights,
       ZAggregator aggregator) {
-    MemberMap unionMembers = null;
-    ScoreSet unionScores = null;
+    MemberMap unionMembers = new MemberMap(keyWeights.size());
```

Comment:
This initial size is a bit misleading, since the number of keys passed to the ZUNIONSTORE command doesn't technically have any relation to the number of members in those keys. There's unfortunately not really a sensible way to estimate an initial size for the `MemberMap` in this method, because even if we use the size of the first set we find, the final size could be anywhere between that size (if none of the other sets contain members not present in the first set) and the combined size of all the sets passed to the command (if all of the sets contain unique members), so some resizing of the `MemberMap` is quite likely even if we use our best guess.

Just for clarity, it might be best to use an initial size of 0 here rather than choosing an arbitrary number.